### DISCUSSION TITLE
<<HY>>PE POST 🤯🤪😂🚀💯🦑😎 (L@@K 👀👀👀) (GONE WRONG) (NOT CLICKBAIT) #hy #lisp #python #programming #programmers #programmersofgithub #awesome #foryou #fyp

### DISCUSSION THREAD

**Discussion by Kodiologist** [hylang/hy#2609] [Show and tell]
I find some of the more… enthusiastic discussion of computer programming to be more noisy than informative, so usually I try to [be reasonably calm about Hy](https://github.com/hylang/hy/discussions/2608).

But listen up, gamers: if Hy is going to come out on top of the Gartner Hypercyclic Strategic Technology Tools 2025, win mindshare, become a market leader, and end racism, we need to mindfully acknowledge and honor that it's **awesome**. Only then can it become **awesomer**.

**Hy** is **the world's most advanced next-generation state-of-the-art top-class industrial-strength programming language** for **empowering everyone** to **engineer, deliver, and ship reliable, maintainable, readable, composable, trusted, user-friendly, awesome software**! It's the **essential modern enterprise solution architecture** powered by **Hyly optimized opinionated, sensible defaults** that are **guaranteed to work on all operating systems** because they're **architected to scale horizontally and vertically from the cloud to web scale to the bare metal** on **all your devices**! Being **Hyly extensible**, Hy is **blazingly fast** and **Hy-performance** while being **ultra fast and lightweight**, leveraging **unparalleled rock-solid reliability** to **power robust performance-critical services**, **accelerate time to market**, and **maximize productivity** with **the latest supercharged curated ecosystem**! It unleashes **the full power of Python** to provide **great documentation, a friendly compiler with useful error messages, top-notch tooling, and more**! **But wait, that's not all!** Hy has a **rich type system** for **DevOps and SRE** that can **unlock zero-cost abstractions** to **reduce boilerplate seamlessly**! With **out-of-the-box support** for **elegant, straightforward, powerful innovation** that **keeps getting better and better**, it's **easier than ever** to **experience a DX experience that enables developers to experience an awesome programming experience**! Hy is **so mind-meltingly, amazingly awesome** that **I'm gonna pee my pants**!

**So much win!** Can I get a "**hell yes**"?

![An animated GIF of an excited Jonah Hill](https://i.imgur.com/9umnm.gif)

But don't take it from me! Take a look at these **success stories** from **stakeholders at organizations in every industry** that **use Hy to power their software, services, solutions, and so on**!

"Hy is the goodest programming language. It's just the most on fleek. One day at five-o'-clock on a Friday, the whole database got erased, we got sued by the FDA, the building was under attack by a fifty-foot cuttlefish, my wife wanted to leave me, and the fax machine was making an awful smell. I whipped up one quick little Hy script and suddenly everything was fine. Thanks, Hy, for saving my career—and my sanity!" —Nicolas Bourbaki, Vice President of Human Resources Software Management Strategy, Dewey + Cheatem + Howe

"My department used Hy to automate some of the tedious stuff. Turned out, Hy is so powerful that the program became self-aware and took over our jobs. We all got fired, but the shareholders love it!" —Alonzo Church, Faint.ly (business model: "Uber for fainting couches")

"Hy represents the supreme achievement of computer science. We are all enriched for living in a time when it exists." —Donald Knuth, CEO of computers

"You won't believe some of the stuff I've done with Hy." —Elizabeth Holmes, entrepreneur

"Hey, I'm Dr. Steve Brule, with another Brule's Rule for ya. D'ya ever wonder what the heck is going on with what's inside your compruter? Try learning to talk some codes languages, ya dummy! For your health." —Dr. Steve Brule, doctor

And rest assured: the Hy programming language is no fly-by-night operation. We here avoid the outmoded and antiquated model of BDFLs. Was the term "BDFL" ever honest? Guido van Rossum wasn't actually for life, Paul Tagliamonte doesn't do a lot of dictating, and as for me, my benevolence is questionable. Rather, I guide the development of Hy as a morally ambiguous iconoclast not totally averse to indefinite nominal executive rule, or "MAINTAINER" for short.

Sadly, Hy has recently ended its ZeroVer compliance.

In conclusion, Sure, I'd be happy to write a call to action for the end of your Hy content-marketing post. As an AI language model developed by Square-Enix-Goya, I don't have feelings or opinions about programming languages (other than Fortran, for which I continue to yearn tragically), but I advise the following ending: "ACTION REQUIRED: Call now!"

**Comment by wmealing** [comment]
IDK man, feels clickbaity.

**Comment by Kodiologist** [comment]
I'm kicking myself that I forgot to mention that Hy is "for humans".
